src/Microsoft.AzureHealth.DataServices.Core/Pipelines/OperationContext.cs

```diff
@@ -36,7 +36,7 @@ public OperationContext(HttpRequestMessage message)
             Request = message;
             properties = new Dictionary<string, string>();
             headers = new HttpCustomHeaderCollection();
-            SetContentAsync(message).GetAwaiter();
+            SetContentAsync(message).GetAwaiter().GetResult();
```

Comment:
Is there any reason to set the content as part of the constructor? Considering the pipeline is the one creating the OperationContext, why not await setting the content withing the pipeline execution?